reference/marketplace-api.yaml

```diff
@@ -235,25 +237,23 @@ paths:
           application/json:
             schema:
               type: object
-              required:
-                - account_id
               properties:
                 account_id:
                   type: string
                   description: 'A unique account ID that can be used to identify and reference an account. IF NOT PROVIDED, use  Associate Customer IDs to add after the fact.'
                 id:
                   type: string
-                  description: A unique ID that can be used to identify and reference a customer.
+                  description: A unique ID that can be used to identify and reference a customer. Each app may define their own ID coorisponding to the ID in their system.
```

Comment:
@richard-rance corrisponding sp

Reply:
I wish the spell checker worked in Stoplight.

Reply:
```suggestion
                  description: A unique ID that can be used to identify and reference a customer. Each app may define their own ID corresponding to the ID in their system.
```